LDAR_Sim/src/utils/generic_functions.py

```diff
@@ -187,11 +187,11 @@ def ecef_to_llh(ecef_km):
     a = 6378.1370
     # Derived Earth semimajor axis (km)
     b = 6356.752314
-    # pythagoras to calculate two earth's minor axises
+    # pythagoras to calculate two earth's minor axes
```

Comment:
Wording suggestion here? While the typo is fixed this comment does not make sense

Reply:
I have made a change, so the PR is now ready for review.